src/Build/Evaluation/LazyItemEvaluator.IncludeOperation.cs

```diff
@@ -167,7 +167,7 @@ class IncludeOperationBuilder : OperationBuilderWithMetadata
             public int ElementOrder { get; set; }
             public string RootDirectory { get; set; }
 
-            public ImmutableList<string>.Builder Excludes { get; } = ImmutableList.CreateBuilder<string>();
+            public ImmutableArray<string>.Builder Excludes { get; } = ImmutableArray.CreateBuilder<string>();
```

Comment:
team triage:
We think this can just be a List. Can you change it to that to avoid the initial allocation cost at ToImmutable time?

Reply:
➡️ Changed to `List<T>` and added a tracking issue to convert back to `ImmutableSegmentedList<T>` once available.